pandas/core/indexes/interval.py

```diff
@@ -142,20 +142,24 @@ class IntervalIndex(IntervalMixin, Index):
 
     Parameters
     ----------
-    data : array-like (1-dimensional)
+    data : array-Like (1-dimensional)
         Array-like containing Interval objects from which to build the
-        IntervalIndex
+        IntervalIndex.
     closed : {'left', 'right', 'both', 'neither'}, default 'right'
```

Comment:
Can you swap the order of `'left'` and `'right'` here (and on the line below), so the default value appears first.  I think some of the other `Interval` related PRs from the sprint did the same thing, and it'd be nice to make this consistent.